solr/webapp/web/partials/query.html

```diff
@@ -229,6 +239,184 @@
           hl.highlightMultiTerm
         </label>
 
+        <label for="hl_q">hl.q</label>
+        <input type="text" ng-model="val['hl.q']" name="hl.q" id="hl_q" value="">
+
+        <label for="hl_qparser">hl.qparser</label>
+        <input type="text" ng-model="val['hl.qparser']" name="hl.qparser" id="hl_qparser" value="">
+
+        <label for="hl_snippets">hl.snippets</label>
+        <input type="text" ng-model="val['hl.snippets']" name="hl.snippets" id="hl_snippets" value="">
+
+        <label for="hl_fragsize">hl.fragsize</label>
+        <input type="text" ng-model="val['hl.fragsize']" name="hl.fragsize" id="hl_fragsize" value="">
+
+        <label for="hl_encoder">hl.encoder</label>
+        <input type="text" ng-model="val['hl.encoder']" name="hl.encoder" id="hl_encoder" value="">
+
+        <label for="hl_maxAnalyzedChars">hl.maxAnalyzedChars</label>
+        <input type="text" ng-model="val['hl.maxAnalyzedChars']" name="hl.maxAnalyzedChars" id="hl_maxAnalyzedChars" value="">
+
+        <label for="hl_method" title="Choose hl.method">
+          hl.method
+          <select ng-model="val['hl.method']" name="hl_method" id="hl_method">
+            <option ng-selected="selected" value=''>------</option>
+            <option value="unified">unified</option>
+            <option value="original">original</option>
+            <option value="fastVector">fastVector</option>
+          </select>
+        </label>
+
+        <p class="clearfix"><a ng-click="showAdvanced=!showAdvanced">
+          <span id="add_advanced" ng-class="{open: showAdvanced}">Show advanced</span></a></p>
```

Comment:
The "Show advanced" expansion does not persist upon refresh (for example when the user execute a query). I think we should persist it otherwise it's pretty annoying to have to click on it every time. But not sure what's the preferred way to do it, should we make the state a param in the url?